src/wp-includes/blocks/file.php

```diff
@@ -96,5 +62,12 @@ function register_block_core_file() {
 			'render_callback' => 'render_block_core_file',
 		)
 	);
+
+	wp_register_script_module(
+		'@wordpress/block-library/file',
+		defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ? gutenberg_url( '/build/interactivity/file.min.js' ) : includes_url( 'blocks/file/view.min.js' ),
+		array( '@wordpress/interactivity' ),
+		defined( 'GUTENBERG_VERSION' ) ? GUTENBERG_VERSION : get_bloginfo( 'version' )
+	);
```

Comment:
Is it possible for GB to deregister the Core modules and re-register it's own versions in the plugin? That would allow us to avoid adding references to `gutenberg_url()` in WP Core. 

Genuine question, not a change request: I haven't kept up to date wiith the script module API.

Reply:
Yes, we still need to figure out that logic. It is still not clear how we're going to do that.